src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.ts

```diff
@@ -136,4 +136,16 @@ export class TokenDetailsStage {
       }
     }
   }
+
+  private isCreatingPartneredDealLike(wizardType: WizardType): boolean {
+    return [WizardType.createPartneredDeal, WizardType.makeAnOffer].includes(wizardType);
+  }
+
+  private isCreatingDealLike(wizardType: WizardType): boolean {
+    return [
+      WizardType.createOpenProposal,
+      WizardType.createPartneredDeal,
+      WizardType.makeAnOffer,
+    ].includes(wizardType);
+  }
```

Comment:
@dkent600 
There are 2+ wizards, that check the deal type.
I was thinking to move such checks to some common place (eventually)?

Note: The changes here are mainly for discussion purposes, ie. to show,
that multiple wizards check for different wizard types.
I can "inline" the methods again too.